### DISCUSSION TITLE
Support for pt.stackoverflow.com

### DISCUSSION THREAD

**Discussion by 13dev** [lowlighter/metrics#489] [Ideas]
Can this metrics support the stackoverflow PT (Portuguese)?

**Comment by lowlighter** [comment]
Doe the Portuguese stackoverflow has its own API endpoint? 
The one used by metrics is `https://api.stackexchange.com/2.2`

**Reply by lowlighter** [reply]
Archiving, but if it does have a different endpoint and a portuguese dev is interested in implementing it, why not
